keyboards/hand88/readme.md

```diff
@@ -0,0 +1,32 @@
+# Hand 88
+
+This is a standard TKL layout PCB. It supports VIA.
+
+* Keyboard Maintainer: [Yiancar](http://yiancar-designs.com/) and on [GitHub](https://github.com/yiancar)
+* Hardware Supported: A TKL keyboard with STM32F072CB
+* Hardware Availability: https://www.hand.engineering 
+
+## Instructions
+
+### Build
+
+Make example for this keyboard (after setting up your build environment):
+
+    make hyprkeys_alpha8x:via
```

Comment:
should this be `make hand88:via` or `make hand88:default`?

Reply:
cheers thanks!

Reply:
now fixed!